components/SummariesHeader.vue

```diff
@@ -0,0 +1,97 @@
+<script setup>
+import { ref, onMounted } from 'vue';
+
+// Initialize reactive references for tracking the count of products, orders, and users.
+const products = ref(0);
+const orders = ref(0);
+const customers = ref(0);
+
+// Asynchronous function to fetch data from API and update state
+const fetchData = async () => {
+
+    try {
+        const responses = await Promise.all([
+            fetch('https://fakestoreapi.com/products'), // Fetch list of products
+            fetch('https://fakestoreapi.com/carts'), // Fetch list of orders
+            fetch('https://fakestoreapi.com/users') // Fetch list of users
+        ]);
+        const data = await Promise.all(responses.map(res => res.json()));
+
+        // Update the state with the count of products, orders, and users
+        products.value = data[0].length;
+        orders.value = data[1].length;
+        customers.value = data[2].length;
+    } catch (error) {
+        console.error("There was an error fetching the data", error);
+    }
+};
+
+onMounted(() => {
+    fetchData();
+});
+</script>
+
+<template>
+    <div class="mt-10 flex justify-center">
+        <div class="stats stats-horizontal max-[340px]:stats-vertical shadow bg-white">
+
+            
+            <div class="stat p-4">
+                <div
+                    class="stat-figure text-primary flex items-center justify-center rounded-full bg-gradient-to-br from-purple-300 to-purple-200 w-14 h-14 hidden md:flex">
+                    <svg xmlns="http://www.w3.org/2000/svg" width="34" height="34" viewBox="0 0 24 24" fill="none"
```

Comment:
SVG can be imported from assets/svg as svg file, or from components/svg/*.vue as SvgComponent